MyFirstProject/LoginAndRegistrationApp/ViewController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ViewController.swift
+//  RegistrationApp
+//
+//  Created by Rishat on 17.09.2020.
+//  Copyright © 2020 Rishat Latypov. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, LoginViewDelegate, RegistrViewDelegate {
+    
+    @IBOutlet weak var statusField: UILabel!
+    @IBOutlet weak var statusView: UIView!
+    
+    var loginAndPasswordDictionary: [String: String] = ["rishatl": "1234"]
+    
+    func validate(login: String, password: String) {
+        if(login == "" && password == ""){
```

Comment:
Скобки без нужды лучше не писать.

```suggestion
        if login == "" && password == "" {
```